lib/rust/ensogl/pack/js/src/runner/index.ts

```diff
@@ -197,14 +197,20 @@ export class App {
      * (like wasm compilation) should be aborted. Currently, we initialize application fully
      * before using stop, but in future we need to support interruption feature. */
     stopped = false
+    accessToken: string | null = null
```

Comment:
`App.accessToken` does not explain what it is for. Also, I'm not sure if EnsoGL Pack should have a notion of an access token. It is meant to be like a web-pack replacement, which allows shipping any EnsoGL Apps easily. So this is probably not the right place to add such things.